packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -0,0 +1,96 @@
+:where(.hop-Popover *) {
```

Comment:
Why are we sitting these on all sub elements? Isn't it redondant? I'd use * with care.

Reply:
I haven't done a precise test, but you can add as many as you like. They are positioned one below the other.